content/post/contributors/yokwejuste.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Yokwejuste"
+date: 2022-01-19T14:53:19+01:00
+images: []
+categories: ["Web development", "Data Science", "Blockchain", "Cloud Computing"]
+tags: ["Open-source", "github"]
+authors: ["yokwejuste"]
+---
+Hi, I'm Yokwejuste, a young inspired blogger, aspiring to be a computer engineer. I like to try everything and being involve in tech 😅.
+The fields in which I'm involved are: Data Science, Web Development, blockchain, Cloud Computing and AI.
```

Comment:
Minor nitpick, why is "blockchain" lower case when everything else is upper case?